openquake/hazardlib/tests/gsim/data/BINDI2011/BINDI2011_MEAN.csv

```diff
@@ -1,673 +1,673 @@
-4.5,0,130,150,MEAN,5,0.0002408192,0.0002886939,0.0003598324,0.0006350925,0.0007061361,0.0007237079,0.0007959762,0.0007707229,0.0006880525,0.0006685249,0.0007077606,0.0007400132,0.0007936502,0.0008679988,0.000873131,0.0008370657,0.0005487817,0.0003202276,0.0001857586,0.0001438595,0.0002539858,0.0290343
-5,0,130,150,MEAN,5,0.0006727914,0.0007929181,0.0009554109,0.0016095201,0.0017908868,0.0018303906,0.0020870838,0.0020835759,0.0018952242,0.0018539053,0.0019337796,0.0020932938,0.0023117069,0.002668944,0.0027414459,0.0026561874,0.0019009856,0.0011655152,0.0007028282,0.0005418252,0.000711247,0.0869623
-5.5,0,130,150,MEAN,5,0.0017167636,0.0019923215,0.0023535356,0.0038180928,0.0042795552,0.0043920095,0.0052051108,0.0053562226,0.0049819867,0.0049631526,0.0051337817,0.0057685958,0.0065914864,0.0080056288,0.0083507518,0.0081707413,0.0063351807,0.0040730831,0.0025480669,0.0019562134,0.0018360296,0.258417
-6,0,130,150,MEAN,5,0.0040012033,0.0045797087,0.0053789826,0.0084780634,0.009635584,0.0099984602,0.0123472338,0.01309387,0.0124980498,0.0128271122,0.0132420347,0.0154864301,0.0183989436,0.0234263484,0.0246788659,0.0243646913,0.0203109115,0.0136673584,0.0088515752,0.0067707321,0.0043690863,0.761888
-6.5,0,130,150,MEAN,5,0.0085178629,0.0096311483,0.0114061377,0.0176216139,0.0204412312,0.0215950401,0.0278586469,0.0304381211,0.0299220427,0.0320042012,0.0331883977,0.0405033319,0.0502739468,0.0668752326,0.070758006,0.0704289436,0.062648305,0.0440345072,0.0294625586,0.02246496,0.0095840883,2.22854
-6.9,0,130,150,MEAN,5,0.0163500278,0.0187114866,0.0218680181,0.0313104883,0.035723922,0.0374600909,0.049475407,0.0547846614,0.0539323826,0.0587123024,0.0594524124,0.0738685484,0.0922569889,0.1260981069,0.1325590288,0.1310957361,0.1230440568,0.0895610632,0.0614411649,0.0463734303,0.0186173668,4.82181
-4,0,160,150,MEAN,5,4.70717319370019E-005,5.44851707769728E-005,6.680364854461E-005,0.0001240413,0.0001428133,0.0001558779,0.0001757562,0.0001667929,0.0001503602,0.0001515614,0.0001668123,0.0001759622,0.000189863,0.0001984266,0.0001996543,0.0001937308,0.0001135352,6.29360688920273E-005,3.55295641223048E-005,2.82136101522946E-005,5.09495087517144E-005,0.00656691
-4.5,0,160,150,MEAN,5,0.0001503969,0.0001712889,0.0001995585,0.0003471257,0.0003957111,0.0004266024,0.0004974624,0.0004867524,0.000444581,0.0004455405,0.0004781194,0.0005204387,0.0005740227,0.0006360765,0.0006565524,0.0006434267,0.0004161972,0.0002433655,0.0001433109,0.0001130886,0.000161521,0.0205051
-5,0,160,150,MEAN,5,0.0004389052,0.0004926382,0.0005530706,0.0009093197,0.0010331153,0.0011076769,0.0013392137,0.0013507875,0.0012545263,0.0012644073,0.0013315148,0.0014995437,0.0016988778,0.0019891094,0.00209
```

Comment:
These values are a too rough approximation. It would be better to use a large number of significant digits.

Reply:
Ciao Marco,
how many significant digits are needed?
We tried to reproduce exactly the existing format for all the changes we made to Bindi et al. (2011). Instead, we reduced the number of significant digits for verification tables of Bindi et al. (2014).

Reply:
There is no specific rule. I would suggest using something like 6 or 7 digits.